content/docs/common-areas/cdn.md

```diff
@@ -0,0 +1,41 @@
+
+By utilizing caching to reduce hosting bandwidth, helping to prevent interruptions in service, and improving security, CDNs are a popular choice to relieve some of the major pain points that come with traditional web hosting.
+
+## Benefits of using a CDN
+
+The benefits of using a CDN depend on the size and needs of an Internet property. The following are the common benefits:
+
+- Improving website load times: web site’s loading time gets shorter because a CDN reuses the sessions previously opened by visitors. In such a way that when a user wants to access the website, it will load quickly and no need to wait to get the full view of the website. In case the caching content is lost they can be recuperated by the interconnection with other CDN servers.
+- Another major concern for websites is the consumption of bandwidth for hosting. The quantity of data that an origin server needs to provide can be reduced by CDNs through caching and other improvements, which lowers hosting costs for website owners.
+- Increasing content availability and redundancy: Large volumes of traffic or hardware issues can prevent a website from operating normally. A CDN can manage more traffic and withstand hardware failures than multiple origin servers because of its distributed architecture.
+- Improving website security: By offering DDoS mitigation, upgrades to security certificates, and other optimizations, a CDN may enhance website security.
+- SEO Improvement: Page loading speed is a very important factor when it comes to page ranking. This is one of the things search engines take into account in their algorithms. Not only that, site speed also increases the frequency with which Google crawls your site's pages, which in turn makes your content more widely available and indexed faster. 
+
+## What Are Some CDN Security Concerns?
+
+Once you understand the answer to “what is CDN security?,” it’s easy to see how CDN security can be vital to users having a great browsing experience. Unfortunately, security for a CDN can come with risks.
+
+Unlike firewalls, CDNs alone are unable to block bad bots from infecting a website. As such, it’s possible to hijack and exploit CDN servers containing cached information in a variety of ways.
+
+For example, if a hacker gained access to data cached on a CDN used by multiple businesses, customers’ private information would become vulnerable. Cybercriminals could then steal passwords, email addresses, and any other sensitive information which could be used to log into private accounts or leveraged in a ransom situation.
+
+Distributed denial-of-service (DDOS) attacks are another method for extortion and blackmailing, making them one of many major CDN security concerns. A simulated test showed that 16 different CDNs were vulnerable to a DDOS exploit that caused servers to repeatedly run the same command. These repeated commands eventually led to the servers becoming overloaded, ultimately taking the content offline.
```

Comment:
Can you share references for this? It seems counter-intuitive to me. CDNs may not help against DDoS but they definitely provide a cushion. What you are saying is the opposite. It feels like this was a specific issue against a specific CDN service and if so, we should reconsider putting it here.

Reply:
This is the source:
https://www.sitelock.com/blog/cdn-security-best-practices/